src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/PrisonerNonAssociations.kt

```diff
@@ -53,7 +53,7 @@ data class PrisonerNonAssociation(
 
   @Schema(description = "Explanation of why prisoners are non-associated", required = true, example = "John and Luke always end up fighting")
   val comment: String,
-  @Schema(description = "User ID of the person who created the non-association. NOTE: For records migrated from NOMIS/Prison API this is free text and may not be a valid User ID. Additionally, migrated records might use an internal system username", required = true, example = "OFF3_GEN")
+  @Schema(description = "User ID of the person who created the non-association. NOTE: For records migrated from NOMIS/Prison API this is free text and may not be a valid User ID. Additionally, migrated records might use an internal system username. It can be an empty string", required = true, example = "OFF3_GEN")
```

Comment:
`updatedBy` can also be empty string now no? See [other comment](https://github.com/ministryofjustice/hmpps-non-associations-api/pull/161/commits/b48118dd48ec850a8e0cb18238bfa04c0389e361#r1378948269).

I think I would prefer if `updatedBy` was always populated even on creation (would match `authorisedBy`) even tho they didn't _really_ update the record. 

But I don't know what you think and I have no strong opinion on this.

Reply:
How does `updatedBy` gets populated now when a record is created?

Reply:
Alright, ignore me, missed this line for some reason 🤦 

https://github.com/ministryofjustice/hmpps-non-associations-api/pull/161/files#diff-718a1a6e743739af98497c6018f0eb75efaa098dbaa28385e7be95cd23563f16R99

Reply:
no, `updatedBy` cannot be null. the one place this DTO is created uses `[some nullable field] ?: ""` so it can indeed be a blank string.